src/signal.rs

```diff
@@ -180,7 +235,11 @@ pub fn create_sigset(signals: &[c_int]) -> errno::Result<sigset_t> {
     Ok(sigset)
 }
 
-/// Retrieves the signal mask of the current thread as a vector of c_ints.
+/// Retrieve the signal mask that is blocked of the current thread.
+///
+/// Use [`pthread_sigmask`](http://man7.org/linux/man-pages/man3/pthread_sigmask.3.html)
+/// to fetch the signal mask which is blocked for the caller, return the signal mask as
+/// a vector of c_int.
```

Comment:
```suggestion
/// a vector of `i32`.
```

Stressing out again my previous point of using either `c_int` or `i32` across the code.